### PR TITLE
SCP-5119 Migrate `marlowe-apps` to use `marlowe-proxy`

### PR DESCRIPTION
**Pull request by jhbertra**
It also adds the `marlowe-client` library for a standard client interface to the runtime.

Pre-submit checklist:
- Branch
    - [x] Commit sequence broadly makes sense
    - [x] Key commits have useful messages
    - [x] Formatting, materialized Nix files, PNG optimization, etc. are updated
- PR
    - [x] Self-reviewed the diff
    - [x] Useful pull request description
        - Review required
        - [x] Substantial changes to code, test, or documentation
    - [x] Reviewer requested

### CODE REVIEW

**Comment by jhbertra** [comment]
@bwbush this is still a draft because I need to update documentation, but please consider the code complete and feel free to test.

File: marlowe-client/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Marlowe Client
+
+A client library for the Marlowe Runtime
+
+TODO
```
Comment (jhbertra) [inline marlowe-client/README.md:5]: TODO

File: marlowe-client/marlowe-client.cabal [hunk 2]
```diff
@@ -0,0 +1,69 @@
+cabal-version: 3.0
+name: marlowe-client
+version: 0.1.0.0
+synopsis:
+  Run integration tests in the context of a marlowe runtime
```
Comment (jhbertra) [inline marlowe-client/marlowe-client.cabal:5]: Change this

**Review by bwbush — COMMENTED** [review]
The code looks good, especially `marlowe-client`, but I'm seeing deserialization errors between `marlowe-scaling` and `marlowe-proxy`. I'll investigate further--it's probably a configuration issue on my end.

**Comment by jhbertra** [comment]
> The code looks good, especially `marlowe-client`, but I'm seeing deserialization errors between `marlowe-scaling` and `marlowe-proxy`. I'll investigate further--it's probably a configuration issue on my end.

Probably - I was able to run it fine on my end.

**Review by bwbush — APPROVED** [review]
Using `marlowe-scaling`, 100% success on 100 clients operating against `marlowe-proxy`, and no change in throughput.

Also, `marlowe-finder` works fine, too.

**Pull request merged**